### PR TITLE
hyperkit v0.20210107

### PR DESCRIPTION
**Pull request by ebmmy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/hyperkit.rb [hunk 1]
```diff
@@ -11,9 +11,8 @@ class Hyperkit < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "f057fe7b3856421d0fdf1df3d9981e0729ee77c27ed3d4cb918e9523f7d5d9be"
   end
 
-  depends_on "aspcud" => :build
   depends_on "ocaml" => :build
-  depends_on "opam" => :build
+  depends_on "opam" => [">=2.0", :build]
```
Comment (carlocab) [inline Formula/hyperkit.rb:15]:
```suggestion
  depends_on "opam" => :build
  depends_on "pkg-config" => :build
```
Specifying the version doesn't do anything.

Comment (ebmmy) [reply Formula/hyperkit.rb]: Thanks didn't know that :)

**Comment by Bo98** [comment]
Oh excellent, this works in our VMs again.

I'm fine with this. We should/could probably use our ocaml-findlib or ocaml-num, but I'm fine with merging this considering it was already like that.

**Review by carlocab — APPROVED** [review]

File: Formula/hyperkit.rb [hunk 2]
```diff
@@ -27,20 +27,19 @@ def install
...
     ENV["PERL5LIB"] = "#{opam_dir}/system/lib/perl5"
     ENV["OCAML_TOPLEVEL_PATH"] = "#{opam_dir}/system/lib/toplevel"
     ENV.prepend_path "PATH", "#{opam_dir}/system/bin"
 
-    system "opam", "config", "exec", "--",
-           "opam", "install", "-y", "uri.3.1.0", "qcow.0.11.0", "conduit.2.1.0", "lwt.5.3.0",
+    system "opam", "exec", "--",
+           "opam", "install", "-y", "uri.4.2.0", "qcow.0.11.0", "conduit.2.1.0", "lwt.5.3.0",
            "qcow-tool.0.11.0", "mirage-block-unix.2.12.0", "conf-libev.4-11", "logs.0.7.0", "fmt.0.8.8",
            "mirage-unix.4.0.0", "prometheus-app.0.7"
```
Comment (Bo98) [inline Formula/hyperkit.rb:38]: `conf-libev`, `fmt`, `mirage-unix` and `prometheus-app` can probably be bumped given upstream don't seem to specify an older version needs to be used.

**Pull request closed without merging**